modules/core/src/main/java/org/apache/ignite/internal/util/IgniteSpinBusyLock.java

```diff
@@ -26,7 +26,7 @@
  *
  * <p>For example, there may be a manager that have different threads for some purposes and the manager must not be stopped while at least a
  * single thread is in "busy" state. In this situation each thread must enter to "busy" state calling method {@link #enterBusy()} in
- * critical pieces of code which, i.e. use grid kernal functionality, notifying that the manager and the whole grid kernal cannot be stopped
+ * critical pieces of code, notifying that the manager and the whole ignite node cannot be stopped
```

Comment:
Why? Seems that IgniteSpingBusyLock is a general-purpose lock, so it's not only about node stopping.

Reply:
I've just removed words about `grid kernal` because this is a legacy from Ignite 2) 

Talking about node stopping, this section is just an example of usage `IgniteSpinBusyLock`, so I think it's useful.

Reply:
ok